Source/WebCore/workers/service/server/SWServerRegistration.cpp

```diff
@@ -423,4 +424,39 @@ std::optional<ExceptionData> SWServerRegistration::setNavigationPreloadHeaderVal
     return { };
 }
 
+void SWServerRegistration::addCookieChangeSubscriptions(Vector<CookieChangeSubscription>&& subscriptions)
+{
+    for (auto& requestedSubscription : subscriptions) {
+        bool isNewSubscription = true;
+        for (auto& currentSubscription : m_cookieChangeSubscriptions) {
```

Comment:
Feels like this loop could be replaced with `WTF::anyOf()`

Reply:
Changed to use HashSet